scipy/stats/_stats_pythran.py

```diff
@@ -178,3 +178,35 @@ def siegelslopes(y, x, method):
         medinter = np.median(y - medslope*x)
 
     return medslope, medinter
+
+
+# pythran export _poisson_binom_pmf(float64[:], int64)
+def _poisson_binom_pmf(p, n):
```

Comment:
`n` is the length of `p`, but I was having trouble getting this working without passing it in explicitly. I think may have been to `args` (passed into `_poisson_binom`) being a view before, so this might not be needed anymore.